libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -393,6 +405,10 @@ const char *AP_RCProtocol::protocol_name_from_protocol(rcprotocol_t protocol)
     case SBUS:
```

Comment:
Line 315 above needs changing

Reply:
Wish I'd seen this comment earlier - cost me several minutes to figure this one out, so I also added a comment warning future-me of this landmine.

Thanks!